### PR TITLE
Image Service Assignment

### PR DESCRIPTION
**Pull request by michalderdak**

### CODE REVIEW

File: image-service/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Part One - Programming
+Create you own micro-service to do the image processing, seperate from the API service. You can choose to write it in whatever programming language you are comfortable with. We expect it to:
+
+* Be a simple HTTP server
+* Take as part of the request:
+    * The image originally sent to the API
+    * Parameters for scaling the image or not
+    * Parameters for grayscaling the image or not
+
```
Comment (Miro001) [inline image-service/README.md:45]: What outputs do we expect

File: image-service/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+### Part Two - Kubernetes
+Create Dockerfile for containerization of your service as well as manifest to deploy your new service to the Minikube. Take an inspiration from already existing manifest and dockerfile for the API.
+
+Deploy your new service onto Minikube. The API should be able to call your service within Minikube. You should be able to call the API through port forward. Useful link: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ 
+
+### Part Three - Optional extras / Questions to think about
+#### Extras
+- Have an option to supply the API with a URL for downloading images, instead of image bytes. Whether you want to download the image content in the API and send bytes to your service - or send the URL and download in the service - is up to you.
+- Implement caching of the images so that we don't do processing of the images twice. It's up to you where and how.
+
```
Comment (Miro001) [inline image-service/README.md:59]: Think about Storing the preprocessed and post-processed  images in DB of your  choice  maybe?

File: image-service/README.md (hunk 1 above)
Comment (Miro001) [reply image-service/README.md:45]: Parameters for iamge type support at least png and jpeg on output

File: image-service/README.md (hunk 2 above)
Comment (michalderdak) [reply image-service/README.md:59]: I was thinking that caching is different from DB/storage that's why I added a question `If we wanted to store the images for further processing - how and where would you store them?`

File: image-service/README.md (hunk 1 above)
Comment (michalderdak) [reply image-service/README.md:45]: we can say the service needs to be able to handle jpegs and pngs ... and than add a question or maybe a task how would you support PDFs?

**Pull request merged**